posts/2023-05-02-23.0.0.4.adoc

```diff
@@ -0,0 +1,194 @@
+[ERROR ] SESN0306E: An exception occurred when invalidating a session in the cache. The exception is: java.lang.RuntimeException: Internal Server Error
+[ERROR ] SESN0304E: An exception occurred when reading in an object of the application data for a session from the cache. The exception is: java.lang.ClassNotFoundException: com.ibm.gs.houston.payment.ach.ibm.forms.ACHCreditForm
+----
++
+This issue has been resolved by introducing the property `appInCacheName` which, when set to `true`, will include the application name in the corresponding cache.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24915[Server hangs at startup when enabling trace specification: com.ibm.ws.*=all]
++
+When the server is started with `com.ibm.ws.*=all` trace specification, the server hangs, because the application was not loaded yet.
++
+The issue has been resolved and the server no longer hangs even when a very broad trace level is set.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24804[Encrypted value for internalClientSecret within oauthProvider does not work]
++
+The `internalClientSecret` and `internalClientId` parameters values within `oauthProvider` configuration of OpenID Connect Server are used to generate tokens for https://openliberty.io/docs/latest/oidc-tools.html#personal[Personal Token Management] and https://openliberty.io/docs/latest/oidc-tools.html#users[Users Token Management] UI pages. However, if the `internalClientSecret` value is encrypted via https://openliberty.io/docs/latest/reference/command/securityUtility-commands.html[`securityUtility` command], the two UI pages no longer work. This is caused by an error that states the `ClientId` and `Secret` are invalid.
++
+This issue has been resolved by adding support for encrypting the `internalClientSecret` value.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24793[JSP Options to pick up web-ext jsp-attribute values on start up (honor disableTldSearch to improve app start up time)]
++
+The JSP Engine does not pick up the `jsp-attributes` during the initial stage of application start-up when set via `ibm-web-ext.xml` or  `server.xml` files.   However, later phases of the start up did pick up the `jsp-attributes`. 
++
+This inadvertently slowed application start up because `disableTldSearch` (which defaults to `true`) would not be registered when set to `false` and thus, the JSP Engine would continue it's time consuming search for TLDs in the various application libraries and other locations. 
++
+The issue has been resolved and the JSP Engine now honors the `jsp-attributes` when set via `ibm-web-ext.xml` or `server.xml` files.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24683[Port MYFACES-4594 ]
++
+MYFACES-4594 was opened because by default `org.apache.myfaces.LOG_WEB_CONTEXT_PARAMS` defaults to `dev-only` which is defined here: https://myfaces.apache.org/#/core40
```

Comment:
```suggestion
MYFACES-4594 was opened because `org.apache.myfaces.LOG_WEB_CONTEXT_PARAMS` defaults to `dev-only`, which is defined here: https://myfaces.apache.org/#/core40
```